src/definitions.jl

```diff
@@ -617,43 +670,60 @@ Base.adjoint(p::AdjointPlan) = p.p
 # always have AdjointPlan inside ScaledPlan.
 Base.adjoint(p::ScaledPlan) = ScaledPlan(p.p', p.scale)
 
+"""
+    size(p::Plan)
```

Comment:
Shouldn't this docstring be attached to `size(::Plan)` instead of `size(::AdjointPlan)`?